staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -237,8 +240,12 @@ func NewReflectorWithOptions(lw ListerWatcher, expectedType interface{}, store S
 		r.expectedGVK = getExpectedGVKFromObject(expectedType)
 	}
 
-	if s := os.Getenv("ENABLE_CLIENT_GO_WATCH_LIST_ALPHA"); len(s) > 0 {
-		r.UseWatchList = true
+	// don't overwrite UseWatchList if already set
+	// because the higher layers (e.g. storage/cacher) disabled it on purpose
+	if r.UseWatchList == nil {
```

Comment:
end-state I'd like to see this entire option go away. Once the client controls streaming list under .List, this should go away.

Reply:
this can go away once the cacher is switched to using streaming from etcd and we know it works.